src/server/common/components/cookie-banner/template.njk

```diff
@@ -0,0 +1,41 @@
+{# Fallback message for users with JavaScript disabled #}
+<noscript>
+  <div class="govuk-cookie-banner" data-nosnippet role="region" aria-label="Cookies on {{ params.serviceName }}">
+    <div class="govuk-cookie-banner__message govuk-width-container">
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+          <h2 class="govuk-cookie-banner__heading govuk-heading-m">Cookies on {{ params.serviceName }}</h2>
+          <div class="govuk-cookie-banner__content">
+            <p class="govuk-body">We use some essential cookies to make this service work.</p>
+            <p class="govuk-body">JavaScript is disabled, so you cannot set cookie preferences. Analytics cookies will not run.</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</noscript>
+
+<div class="govuk-cookie-banner" data-nosnippet role="region" aria-label="Cookies on {{ params.serviceName }}" id="cookie-banner" data-cookie-name="{{ params.cookieName }}" data-expiry-days="{{ params.expiryDays }}" data-ga-tracking-id="{{ params.gaTrackingId }}" hidden>
```

Comment:
Can we not use the official GOV.UK Design System Cookie Banner component than hardcoding the HTML? https://design-system.service.gov.uk/components/cookie-banner/